client/containers/Pub/PubSubmissionHeader/themes.scss

```diff
@@ -0,0 +1,269 @@
+$mobile-viewport-cutoff: 750px;
```

Comment:
This duplicated file is full of SCSS mixins, which you can think of as like compile-time CSS functions. And like a directory full of functions, it'd be most sensible to

```scss
@import "../PubHeader/themes.scss";
```

from `spubHeader.scss` rather than duplicating them here.